### PR TITLE
use new watson conversation name, merge after march 15

### PR DESCRIPTION
**Pull request by stevemar**

### CODE REVIEW

File: README-ko.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 *다른 언어로 보기: [English](README.md).*
 
-이번 개발 과정에서는 Watson [Speech-to-Text](https://www.ibm.com/watson/developercloud/speech-to-text.html) 와 Watson  [Assistant](https://www.ibm.com/watson/developercloud/Assistant.html) 서비스를 기반으로 VR(Virtual Reality, 가상 현실) 게임을 만들어보겠습니다.
+이번 개발 과정에서는 Watson [Speech-to-Text](https://www.ibm.com/watson/developercloud/speech-to-text.html) 와 Watson  [Assistant](https://www.ibm.com/watson/developercloud/conversation.html) 서비스를 기반으로 VR(Virtual Reality, 가상 현실) 게임을 만들어보겠습니다.
```
Comment (scottdangelo) [inline README-ko.md]: This is a URL and cannot be changed.

File: README-ko.md [hunk 2]
```diff
@@ -51,10 +51,10 @@ Watson 앱을 한 레벨 위로 끌어올리고 싶으신가요? 아니면 Watso
 [IBM Cloud](https://console.ng.bluemix.net/)에서:
 
 1. [Speech-To-Text](https://console.ng.bluemix.net/catalog/speech-to-text/) 서비스 인스턴스를 생성합니다.
-2. [Conversation](https://console.ng.bluemix.net/catalog/services/conversation/) 서비스 인스턴스를 생성합니다.
-3. 대시보드에 서비스가 표시되면 생성된 Conversation 서비스를 선택하고 !["Launch Tool"](/doc/source/images/workspace_launch.png?raw=true) 버튼을 클릭합니다.
-4. Conversation Tool에 로그인한 후 !["Import"](/doc/source/images/import_icon.png?raw=true) 단추를 클릭합니다.
-5. 이 저장소 복제본에 있는 Conversation  [`workspace.json`](data/workspace.json) 파일을 가져옵니다.
+2. [Assistant](https://console.ng.bluemix.net/catalog/services/Assistant/) 서비스 인스턴스를 생성합니다.
```
Comment (scottdangelo) [inline README-ko.md]: URL

File: README-pt.md [hunk 3]
```diff
@@ -31,20 +31,20 @@ Na sua máquina local:
 2. `cd vr-speech-sandbox-vive`
 No [Bluemix](https://console.ng.bluemix.net/):
 1. Crie uma instância de serviço do [Speech-To-Text](https://console.ng.bluemix.net/catalog/speech-to-text/).
-2. Crie uma instância de serviço do [Conversation](https://console.ng.bluemix.net/catalog/services/conversation/).
-3. Quando vir os serviços no painel, selecione o serviço Conversation que foi criado e clique no botão !["Launch Tool"](/doc/source/images/workspace_launch.png?raw=true).
-4. Depois de efetuar login na Conversation Tool, clique no botão !["Import"](/doc/source/images/import_icon.png?raw=true).
-5. Importe o arquivo do Conversation [`workspace.json`](data/workspace.json) localizado no seu clone deste repositório. 
+2. Crie uma instância de serviço do [Assistant](https://console.ng.bluemix.net/catalog/services/Assistant/).
```
Comment (scottdangelo) [inline README-pt.md]: Second is a URL

**Review by scottdangelo — CHANGES_REQUESTED** [review]

File: README-pt.md [hunk 4]
```diff
@@ -31,20 +31,20 @@ Na sua máquina local:
...
+6. Siga [estas instruções](https://github.com/watson-developer-cloud/unity-sdk#configuring-your-service-credentials) para incluir suas credenciais dos serviços Speech To Text e Assistant (localizadas no [IBM Bluemix](https://console.ng.bluemix.net/)). 
 7. Selecione `Advanced Mode` na janela de configuração.
-8. Clique em `Add Variable` e nomeie sua nova variável `ConversationV1_ID` e, em seguida, defina seu valor como o ID da área de trabalho do Conversation. ![Variable Configuration Example](doc/source/images/add_variable.png?raw=true) Para localizar o ID da área de trabalho, selecione o menu de expansão na área de trabalho do Conversation e selecione `View details`. 
+8. Clique em `Add Variable` e nomeie sua nova variável `AssistantV1_ID` e, em seguida, defina seu valor como o ID da área de trabalho do Assistant. ![Variable Configuration Example](doc/source/images/add_variable.png?raw=true) Para localizar o ID da área de trabalho, selecione o menu de expansão na área de trabalho do Assistant e selecione `View details`.
```
Comment (stevemar) [inline README-pt.md]: i'm not sure about this one, @scottdangelo ?

Comment (scottdangelo) [reply README-pt.md]: Yes, this is a comment about a variable and cannot be changed.

**Review by scottdangelo — APPROVED** [review]

**Pull request merged**